### PR TITLE
charts: Remove chart

### PR DESCRIPTION
**Pull request by mrueg**
**What this PR does / why we need it**:
Removes the deprecated helm chart and adds a note to the README about its new location.

Ideally get this in before we branch out to release-2.1

@tariq1890 @lilic

### CODE REVIEW

**Review by lilic — COMMENTED** [review]
/lgtm

Thank you for doing this!

/hold 
until @tariq1890 has a look.

File: README.md [hunk 1]
```diff
@@ -320,6 +321,11 @@ spec:
 
 For the full list of arguments available, see the documentation in [docs/cli-arguments.md](./docs/cli-arguments.md)
 
+
+#### Helm Chart
+
+A helm chart is maintained in [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics).
```
Comment (tariq1890) [inline README.md:327]: Should we state that it is if the `official helm chart` for `kube-state-metrics`?

Comment (mrueg) [reply README.md]: good idea! I updated it.

**Review by tariq1890 — APPROVED** [review]
/hold cancel

Thanks @mrueg!

**Pull request merged**